### PR TITLE
Chat Controller && App

### PR DESCRIPTION
**Pull request by vphilippon**
@grondinjc @maym2104 Ready for review.

### CODE REVIEW

File: src/app/python/chat.py [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+    """
+    Handle the message. Return the data to send, and to who.
+
+    @param author: The author of the message
+    @param message: The message received
+
+    @return tuple (<<Data to send>>, <<Users to send to>>)
+    """
+    self._logger.debug("Handling message from {0}, message: {1}".format(author, message))
+    data = {"author": author, "message": message.strip()}
```
Comment (grondinjc) [inline src/app/python/chat.py]: @vphilippon : The following dictionnary appears many times:
 {"author": someauthor, "message": somemsg} 

Could it be refactored into a method that return the dictionnary ?
Therefore, changing the structure params would be done at only one place.

Comment (grondinjc) [inline src/app/python/chat.py]: Example, adding the timestamp would be done in that function

File: src/server/chatController.py [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+      }
+
+    Output on the WS will be JSON of the following format:
+      {
+        'author':  '<<Author (username) of the sender>>',
+        'message': '<<Content of the message>>'
+      }
+    """
+    if not cherrypy.session.get('username'):
+      cherrypy.session['username'] = str(uuid.uuid4())  # XXX Session should be set by the id module
```
Comment (grondinjc) [inline src/server/chatController.py:86]: This code appears many times.
It will be reviewed when integrating identification/authentication
:+1:

**Comment by vphilippon** [comment]
@grondinjc Change made

**Comment by grondinjc** [comment]
:+1:

**Pull request merged**